packages/osl-operator-bundle-image/resources/scripts/add_annotations.py

```diff
@@ -41,7 +41,7 @@ def merge_annotations():
         # Add the source annotations to the end of destination annotations
         dest_annotations[key] = value
 
-    last_key = list(dest_annotations.keys())[-1]
+    last_key = list(dest_annotations.keys())[-2]
```

Comment:
This change here is because of this two below?

  operators.operatorframework.io.bundle.channels.v1: stable
  operators.operatorframework.io.bundle.channel.default.v1: stable